src/Material/Table.elm

```diff
@@ -248,9 +246,10 @@ type Order
 
 {-| Dispatch given message when cell is clicked. 
 -}
-onClick : m -> Property { a | onClick : Maybe (Attribute m) } m
+onClick : m -> Property c m
```

Comment:
Again, we should consider getting rid of all these and make `Options.onClick` the canonical thing you use.